simple/stats/config.md

```diff
@@ -36,7 +36,16 @@ The config parameters for the files to be imported should be specified in a `con
         "Provenance2 Name": "http://source1.com/provenance2"
       }
     }
-  }
+  },
+  "database": {
+    "type": "cloudMySql",
```

Comment:
nit: [Cloud SQL](https://cloud.google.com/sql)

Reply:
Cloud SQL supports postgresql and sql server in addition to mysql. We need to explicitly add support for them if we want to support them.

Reply:
From user perspective, there is no difference in this config and data preparation? They seem to be internal implementation details which user does not need to be aware of.

Reply:
I've updated it to `cloudsql` for now but configs will likely be different for different DBs. We can revisit this if we have customers that want support for other cloud sql dbs.